privacyidea/lib/error.py

```diff
@@ -33,6 +33,21 @@
 log = logging.getLogger(__name__)
 
 
+class ERROR:
+    SUBSCRIPTION = 101
+    TOKENADMIN = 301
+    CONFIGADMIN = 302
+    POLICY = 303
+    VALIDATE = 401
+    REGISTRATION = 402
+    CA = 503
+    HSM = 707
+    SELFSERVICE = 807
+    SERVER = 903
+    USER = 904
+    PARAMETER = 905
```

Comment:
One question: Do I understand correctly that these do not correspond to HTTP error codes, these are just the codes used in the privacyIDEA response?

Reply:
Yes. These are error codes in the JSON response

    { status: False,
      error: { code: ...,
                  message: .... }